### PR TITLE
OP-471 oh gui vaccine window error

### PR DESCRIPTION
**Pull request by mwithi**
See OP-471.

Also changed the whole layout and improved the model

### CODE REVIEW

File: src/main/java/org/isf/patvac/gui/PatVacEdit.java [hunk 1]
```diff
@@ -58,6 +56,11 @@
 import org.isf.vaccine.model.Vaccine;
 import org.isf.vactype.manager.VaccineTypeBrowserManager;
 import org.isf.vactype.model.VaccineType;
+import java.awt.BorderLayout;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/patvac/gui/PatVacEdit.java:59]: These imports are out of order; they should be nearer to line 26

Comment (mwithi) [reply src/main/java/org/isf/patvac/gui/PatVacEdit.java]: Thanks!

File: src/main/java/org/isf/patvac/gui/PatVacEdit.java [hunk 2]
```diff
@@ -513,27 +536,29 @@ private void setPatient(Patient selectedPatient) {
...
+			patientComboBox = new JComboBox<Object>();
+		}
+		patientComboBox.addItem(MessageBundle.getMessage("angal.patvac.selectapatient"));
 		Patient patSelected = null;
-		PatientBrowserManager patBrowser = Context.getApplicationContext().getBean(PatientBrowserManager.class);
 
 		if (GeneralData.ENHANCEDSEARCH){
 			try {
-				pat = patBrowser.getPatientsByOneOfFieldsLike(regExp);
+				patientList = patientManager.getPatientsByOneOfFieldsLike(regExp);
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/patvac/gui/PatVacEdit.java:548]: should probably be spaces around the "catch"

Comment (mwithi) [reply src/main/java/org/isf/patvac/gui/PatVacEdit.java:548]: Done. Found only one...

**Pull request merged**